.github/workflows/generate-event-pdfs.yml

```diff
@@ -40,8 +66,69 @@ jobs:
           # A variables file to use
           variables_file: sorse-variables.txt
 
+      - name: Move PDFs to downloads folder
+        if: ${{ env.GENERATE_PDFS == '1' }}
+        run: |
+          if [[ ${GITHUB_HEAD_REF} == '' ]]; then
+            rm downloads/*
+          fi
+          mv _events/*/*.pdf downloads/
+
+      - name: Checkout New Branch
+        if: ${{ env.GENERATE_PDFS && env.GITHUB_HEAD_REF == '' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.TEMP_TOKEN }}
+          SOURCE_BRANCH: ${{ github.head_ref }}
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          printf "GitHub Actor: ${GITHUB_ACTOR}\n"
+          export BRANCH_FROM="add-pdfs/${SOURCE_BRANCH}"
+          git remote set-url origin "https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git"
+
+          # Test if branch exists in remote
+          BRANCH_EXISTS=$(git ls-remote --heads origin ${BRANCH_FROM})
+
+          # Force-create the new branch
+          git branch
+          git checkout -B "${BRANCH_FROM}"
+          git branch
+
+          git config --global user.name "github-actions"
+          git config --global user.email "github-actions@users.noreply.github.com"
+
+          git status
+          git add downloads
+
+          if git diff-index --quiet HEAD --; then
+             printf "No changes\n"
+             export OPEN_PULL_REQUEST=0
+          else
+             if [[ -z ${BRANCH_EXISTS} ]]; then
+               export OPEN_PULL_REQUEST=1
+             fi
+             printf "Changes\n"
+             git commit -m "Automated deployment to generate PDFs $(date '+%Y-%m-%d')"
+             git push --force origin "${BRANCH_FROM}"
+          fi
+          echo "::set-env name=OPEN_PULL_REQUEST::${OPEN_PULL_REQUEST}"
+          echo "::set-env name=PULL_REQUEST_FROM_BRANCH::${BRANCH_FROM}"
+          echo "::set-env name=PULL_REQUEST_TITLE::[generate-pdfs] ${SOURCE_BRANCH}"
+          if [[ -z ${PR_NUMBER} ]]; then
+              echo "::set-env name=PULL_REQUEST_BODY::Automated generation of event PDFs for branch ${SOURCE_BRANCH}."
+          else
+              echo "::set-env name=PULL_REQUEST_BODY::Automated generation of event PDFs for branch ${SOURCE_BRANCH}, see #${PR_NUMBER}."
+          fi
```

Comment:
```suggestion
          PR_HINT="<sup>This PR has been automatically opened because some SORSE events changed. Do not manually push to the ${BRANCH_FROM} branch, as these commits might be overwritten. You can close this PR and, as long you do not delete the ${BRANCH_FROM} branch, no new PR will be created.</sup>"
          if [[ -z ${PR_NUMBER} ]]; then
              echo "::set-env name=PULL_REQUEST_BODY::Automated generation of event PDFs for branch ${SOURCE_BRANCH}.\n\n${PR_HINT}"
          else
              echo "::set-env name=PULL_REQUEST_BODY::Automated generation of event PDFs for branch ${SOURCE_BRANCH}, see #${PR_NUMBER}.\n\n${PR_HINT}"
          fi
```